CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 A quick bugfix to resolve a regression
 
+### cli
```

Comment:
you can revert this - we generate this file automatically when we do a release. Also, `8.0.1` is now listed twice